src/styles/block/card.scss

```
@@ -0,0 +1,94 @@
+.card {
+  width: 200px;
+  border: 1px solid #e5e5e5;
+  border-radius: 5px;
+  background-color: #fff;
```

don't forget to create variables for all colors
```suggestion
  border: 1px solid $c-someColor;
  border-radius: 5px;
  background-color: $c-white;
```